pkg/metrics/backup.go

```diff
@@ -214,3 +217,28 @@ func (rm RestoreM) DecreaseRemainingBytes(clusterID uuid.UUID, location backupsp
 	}
 	rm.remainingBytes.With(l).Sub(float64(restoredBytes))
 }
+
+// RestoreState is the enum that defines how node is used during the restore
+type RestoreState int
+
+const (
+	// RestoreStateIdle defines idle state
+	RestoreStateIdle RestoreState = iota
+	// RestoreStateDownloading means that node is downloading data from backup location
+	RestoreStateDownloading
+	// RestoreStateLoading means that node is calling load&stream
+	RestoreStateLoading
+	// RestoreStateError means that node ended up with error
+	RestoreStateError
+)
+
+// SetRestoreState sets restore "state" metric.
+func (rm RestoreM) SetRestoreState(clusterID uuid.UUID, location backupspec.Location, snapshotTag, host string, state RestoreState) {
+	l := prometheus.Labels{
+		"cluster":      clusterID.String(),
+		"location":     location.String(),
+		"snapshot_tag": snapshotTag,
```

Comment:
Are those labels useful here?

Reply:
See this comment https://github.com/scylladb/scylla-manager/pull/3331#discussion_r1146788445